step.yml

```diff
@@ -131,7 +131,7 @@ inputs:
     value_options:
     - "yes"
     - "no"
-- clone_depth:
+- clone_depth: -1
```

Comment:
@ofalvai I'm not sure how best to handle the situation when a user doesn't provide a value for `clone_depth`. From what I've gathered, if no value is specified here in the yaml, it will default to a value of `0` when we parse the config. The problem is, `0` is a valid value that someone could specify, so there'd be no way to determine if it was set to `0` by the user, or if it was set to that because the user provided no value. Unless you know of a way to do that?

Setting this to `-1` works sort of, but it's not really ideal.

Reply:
_DOH_. I was assuming `--depth=0` was the shallow clone value for some reason, but I didn't actually try it 🤦. Never mind then, using `0` as the default value makes perfect sense. 👍 I'll update things.